docs/cloudfoundry/cfctlr-app-install.rst

```diff
@@ -1,104 +1,153 @@
+.. index::
+   single: Cloud Foundry; BIG-IP Controller; deploy
+   single: Cloud Foundry; BIG-IP Controller; Application manifest
+   single: Cloud Foundry; BIG-IP Controller; global configuration
+
 .. _deploy-cf-ctlr:
 
 Deploy the BIG-IP Controller for Cloud Foundry
 ==============================================
 
-Complete the steps provided below to deploy the |cf-long| in `Cloud Foundry`_ or `Pivotal Cloud Foundry`_ using an `Application Manifest`_.
+Complete the steps provided below to deploy the |cf-long| using the default global configuration mode.
+
+.. note::
+
+   If you need to configure BIG-IP objects for individual Routes, you'll need to register the |cfctlr| as a `Service Broker`_. See :ref:`cfctlr per-route vs` for more information.
 
 .. _cf-deployment-prereqs:
 
 Before you begin
 ----------------
 
-- `Enable Docker in Cloud Foundry`_ :fonticon:`fa fa-external-link`.
-- `Create a read-only Admin user in Cloud Foundry`_ :fonticon:`fa fa-external-link` (needs "routing.routes.read" and "routing.router_groups.read" permissions).
-- `Add the BIG-IP device to Cloud Foundry`_ :fonticon:`fa fa-external-link` as a custom load balancer.
+#. `Enable Docker in Cloud Foundry`_ :fonticon:`fa fa-external-link`.
+#. `Create a read-only Admin user in Cloud Foundry`_ :fonticon:`fa fa-external-link`
+
+   This user needs the "routing.routes.read" and "routing.router_groups.read" permissions.
```

Comment:
In service broker mode an additional permissions of 'cloud_controller.read' will be necessary.